locales/en.json

```diff
@@ -1,7 +1,12 @@
 {
   "Polaris": {
-    "Actions": {
-      "moreActions": "More actions"
+    "ActionMenu": {
+      "Actions": {
+        "moreActions": "More actions"
```

Comment:
I'm struggling to distinguish the use-cases between More Actions and RollupActions.

I can see that non-primaryActions passed to a `Page` are rendered in a `ActionGroup`, and when the `Page` drops to smaller breakpoints, these actions are hidden behind the `RollupMenu` with the "View actions" label. 

What causes the `More Actions` label to show? Is that visible, or is another aria-label?

Reply:
I am also unsure of where `More actions` appears but it's only used inside the `Actions` component of `ActionMenu` so I wanted to group them together